syncer/job.go

```diff
@@ -64,9 +65,8 @@ type job struct {
 	// ddl in ShardOptimistic and ShardPessimistic will only affect one table at one time but for normal node
 	// we don't have this limit. So we should update multi tables in normal mode.
 	// sql example: drop table `s1`.`t1`, `s2`.`t2`.
-	sourceTbl       map[string][]string
-	targetSchema    string
-	targetTable     string
+	sourceTbls      map[string][]string
```

Comment:
Seems change this type to `map[string].filter.Table` can make the code a little simpler

Reply:
Should be `map[string]*filter.Table` --> `map[string][]*filter.Table`.

Reply:
And I have thought change it to `[]*filter.Table`, but dismiss it because it will affect performance in https://github.com/pingcap/dm/blob/master/syncer/syncer.go#L1032-L1051

Reply:
I don't understand why it will hurt performance since L1032-1051 also need filter.Table instead of separate shcema/table name, with this change we can avoid newing the structs

Reply:
I mean `change it to []*filter.Table` will hurt performance, because it can't directly `continue` all `len(sourceSchema) == 0`.

Reply:
i also think `map[string]*filter.Table` is enough here, not sure why we need to use `map[string][]*filter.Table`

Reply:
Because one `Schema` may map lots of `Table`s.

Reply:
i see, we need use `map[string][]*filter.Table` to handle `multi schema change` ddl